bitnami/influxdb/templates/_helpers.tpl

```diff
@@ -109,3 +109,32 @@ Get the InfluxDB&trade; initialization scripts secret.
 {{- define "influxdb.initdbScriptsSecret" -}}
 {{- printf "%s" (tpl .Values.influxdb.initdbScriptsSecret $) -}}
 {{- end -}}
+
+{{- /*
+Helper template for determining which persistence settings to use for backup PVC.
+Allows backwards-compatibility for users that have not defined a backups.persistence dict in their values files.
+*/}}
+{{- define "influxdb.backup.pvc" }}
+{{- $ := index . 0 }}
+{{- $persistence := index . 1 }}
+kind: PersistentVolumeClaim
```

Comment:
If possible, I would try to avoid including the resource definition in the helper. But please, let's clarify first my other comment.

Reply:
I'm not quite sure where I should put the template for this. I moved it to a template function because I couldn't figure out a way to set a common variable for rendering the template in the `.yaml` file itself, and `_helpers.tpl` was the only template at the time. I realize that this function doesn't fit the established pattern in there; should I move it to its own `.tpl` file?

Reply:
I would say that the proper place is in `pvc-backup.yaml` and the values for the fields would be typically set with [`coalesce`](https://helm.sh/docs/chart_template_guide/function_list/#coalesce) for example to keep compatibility. But in this case, the scenario might be more complex though...